iceoryx_utils/include/iceoryx_utils/posix_wrapper/timer.hpp

```diff
@@ -321,6 +321,58 @@ class Timer
 };
 
 } // namespace posix
+
+
+namespace clock
+{
+/// @brief This offers the deadline timer functionality. It has user convenient methods to reset the timer [by default
+/// it uses the intialized duration], reset timer to a customized duration, check if the timer is active and user can
+/// also get to know about the remaining time before the timer goes off
+/// @note Introduced as an alternate timer functionality for the deprecated POSIX timers
```

Comment:
I think the note can be removed. It makes sense as comment in the PR description, but not it the code itself. It's also not a full alternative since it's just refactoring out one functionality of the posix timer.

Reply:
Okay removed `@note section`